libr/core/cbin.c

```diff
@@ -2904,7 +2904,7 @@ static void bin_pe_versioninfo(RCore *r, int mode) {
 		} else {
 			r_cons_printf ("# VS_FIXEDFILEINFO\n\n");
 		}
-		char path_fixedfileinfo[256] = R_EMPTY;
+		char path_fixedfileinfo[256 + 16] = R_EMPTY;
```

Comment:
just use r_str_newf, dont use hardcoded non-null-terminated-stack-based-string-holders please :P

Reply:
i'm just the warning-fixer 😅 but ok